scripts/__dropins__/storefront-auth/api/initialize/initialize.d.ts

```diff
@@ -1,5 +1,5 @@
-import { Initializer, Model } from '@dropins/tools/types/elsie/src/lib';
-import { Lang } from '@dropins/tools/types/elsie/src/i18n';
+import { Initializer, Model } from '../../../@adobe-commerce/elsie/src/lib';
```

Comment:
what's with all the changes from absolute to relative paths?

Reply:
@sirugh @loginesta Seems like bug in publishing. Checking this.

Reply:
@fnhipster Need your review on this before I merge.